### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by boo-shehab**
#Hi reviewer 👋 

in this branch, I did all the requirements: 
- I used linters
- I create 3 main files index.html, index.css, and index.js
- I used module and made a few files inside it and use them in my project 
- I Installed Luxon and use it

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @boo-shehab 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a proper readme, please use the microverse [readme](https://github.com/microverseinc/readme-template) template
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by boo-shehab** [comment]
@prolajumokeoni Hi 👋😊
Sorry about that and thanks for counted as invalid I forgot about it but I saw no one review it so I added on my main branch

File: modules/showContent.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      booksList.style.display = 'none';
+      form.style.display = 'none';
+      contact.style.display = 'block';
+      break;
+    default:
+      booksList.style.display = 'block';
+      form.style.display = 'none';
+      contact.style.display = 'none';
+  }
+}
```
Comment (tugsem) [inline modules/showContent.js]: - [ ] Kindly use arrow syntax for the function here to apply ES6 features.

File: modules/booksCards.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+submitButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (bookTitle.value.trim() !== '' && author.value.trim() !== '') {
+    const list = {
+      title: bookTitle.value,
+      author: author.value,
+    };
+    books.add(list);
+    books.setLocalStorage();
+    books.displaybooks(books);
```
Comment (tugsem) [inline modules/booksCards.js:20]: - [ ] I suggest you to clear input fields after submission as it can increase user experience.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @boo-shehab , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️No Linters error
✔️Correct Gihub flow
✔️You documented your work in a professional way

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @boo-shehab, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**